src/screen.cpp

```diff
@@ -472,7 +485,7 @@ bool Screen::cursorPosCallbackEvent(double x, double y) {
     Vector2i p((int) x, (int) y);
 
 #if defined(_WIN32) || defined(__linux__)
-    p /= mPixelRatio;
+    p = (p.cast<float>() / mPixelRatio).cast<int>();
```

Comment:
does it make sense to "round"?  `p = ((p.cast<float>() / mPixelRatio) + 0.5f).cast<int>();`

I'm not sure I understand the impact of fractional computations since it's getting turned into an `int` anyway...

Reply:
So the rounding in the end (`.cast<int>()`) is necessary, as Eigen requires to explicitly convert between the different types and `p` is an int-vector. Does that answer your question?